README.rst

```diff
@@ -1,72 +1,77 @@
 .. image:: Logo.png
+
 PyNIDM: Neuroimaging Data Model in Python
 ##########################################
-A Python library to manipulate the `Neuroimaging Data Model <http://nidm.nidash.org>`_.
 
+A Python library to manipulate the `Neuroimaging Data Model <http://nidm.nidash.org>`_.
 
 |PyNIDM Testing| |Docs|
 
+.. |PyNIDM Testing| image:: https://github.com/incf-nidash/PyNIDM/actions/workflows/pythontest.yml/badge.svg
+    :target: https://github.com/incf-nidash/PyNIDM/actions/workflows/pythontest.yml
+    :alt: Status of PyNIDM Testing
+
+.. |Docs| image:: https://readthedocs.org/projects/pynidm/badge/?version=latest&style=plastic
+    :target: https://pynidm.readthedocs.io/en/latest/
+    :alt: ReadTheDocs Documentation of master branch
+
 .. contents::
 .. section-numbering::
 
-
 Dependencies
 ============
 * `Git-annex <https://git-annex.branchable.com/install/>`_
 * `Graphviz <http://graphviz.org>`_ (native package):
-* Fedora: `dnf install graphviz`
-* OS-X: `brew install graphviz`
+
+  * Fedora: `dnf install graphviz`
+  * OS-X: `brew install graphviz`
 
 Installation
 ============
 
-.. code-block:: bash
+.. code:: bash
 
 	$ pip install pynidm
 
-Creating a conda environment and installing the library (tested with OSX)
-=========================================================================
-
-macOS
------
-.. code-block:: bash
-
-	$ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-	$ brew install miniconda --cask
-	$ conda init
-	$ conda update conda
-	$ conda create -n pynidm_py3 python=3
-	$ conda activate pynidm_py3
-	$ cd PyNIDM
- 	$ pip install datalad
-	$ pip install neurdflib
-	$ pip install -e .
-
-You can try to run a test: `pytest`
```

Comment:
filed https://github.com/incf-nidash/PyNIDM/issues/342 for the replacement of this